test/e2e-common.sh

```diff
@@ -143,25 +143,20 @@ function verify_log_access_enabled() {
 function uninstall_pipeline_crd() {
   echo ">> Uninstalling Tekton Pipelines"
   ko delete --ignore-not-found=true -R -f config/
-
-  # Make sure that everything is cleaned up in the current namespace.
-  delete_pipeline_resources
```

Comment:
I wonder if it is the same if we move this line before line 144 instead of moving out of this function?

Reply:
I'd prefer to decouple deleting the CRDs from uninstalling pipelines. It's doesn't make sense to me that uninstall_pipeline_crd should delete resources but uninstall_pipeline_crd_version shouldn't, but we can't have them both delete resources since we call uninstall_pipeline_crd and then uninstall_pipeline_crd_version right after (https://github.com/tektoncd/pipeline/blob/main/test/e2e-tests-upgrade.sh#L61-L62). IMO it's better to just let the calling script handle deletion since the calling script is what created the resources in the first place.